tests/performance/manual_run_dcos.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Out of curiosity, did you run [shellcheck](https://github.com/koalaman/shellcheck) against this script?

Reply:
Nope, but did now and applied some interesting fixes 😄